src/toggle.js

```diff
@@ -15,8 +15,10 @@ export default class extends Controller {
   }
 
   openValueChanged() {
-    this.toggleableTargets.forEach(target => {
-      target.classList.toggle(this.toggleClass)
-    })
+    if (!!this.toggleClass) {
```

Comment:
Can we turn this into a guard clause instead? Keeps the rest of the method cleaner without the nesting.

`if (!this.toggleClass) { return }`

Reply:
@excid3 done =)

Reply:
That should be a single `!` not the double when it's a guard clause. I think that should fix the failing test. 👍

Reply:
@excid3 oops :D